include/CLI/Option.hpp

```diff
@@ -50,6 +50,18 @@ enum class MultiOptionPolicy : char {
     Reverse,    //!< take only the last Expected number of arguments in reverse order
 };
 
+/// @brief  enumeration for the callback priority
+enum class CallbackPriority : std::uint8_t {
+    PreReadConfigPreHelp = 0,
```

Comment:
```suggestion
    First = 0,
```

Would it be clearer to just call this first

Reply:
Than, for the sake of consistency, i would suggest
```suggestion
    FirstPreHelp = 0,
```
and ...